### PR TITLE
Stabilization of imaging_browser tests that were randomly failing.

### PR DESCRIPTION
**Pull request by nicolasbrossard**
Added helper methods in LorisIntegrationTest and used them in imaging_browserTest in order to stop the random failing of these tests in Travis. Also contains a bug fix: the "Back" link on the imaging browser result page was broken.

### CODE REVIEW

File: modules/imaging_browser/test/imaging_browserTest.php [hunk 1]
```diff
@@ -160,13 +160,14 @@ public function setUp()
             )
         );
 
+        // @codingStandardsIgnoreStart
```
Comment (driusan) [inline modules/imaging_browser/test/imaging_browserTest.php]: can you remove this?

Comment (nicolasbrossard) [inline modules/imaging_browser/test/imaging_browserTest.php]: I am afraid I can't.Phpcs complains about the long file name (value for key 'File') and I don't see a way to satisfy all of its requirements. Open to other suggestions...

Comment (xlecours) [inline modules/imaging_browser/test/imaging_browserTest.php]: phpcbf suggest this :
```
             'File'                  => 'assembly/506145/V1/mri/native/' .
              'loris-MRI_506145_V1_t2_001.mnc',
             'SeriesUID'             => '1.3.12.2.1107.5.2.32.35049.' .
               '2014021711090977356751313.0.0.0',

```

**Comment by nicolasbrossard** [comment]
I modified the code so that the directives to ignore the coding standards were not needed anymore.

**Pull request merged**